docs/sources/write/links/_index.md

```diff
@@ -19,23 +19,40 @@ weight: 600
 
 # Links
 
-Choose your link type based on the applicable scenario:
+Choose your link type based on your goal:
 
 - [Link from source content that's used in multiple projects](#link-from-source-content-thats-used-in-multiple-projects)
 - [Link to `grafana.com` pages](#link-to-grafanacom-pages)
 - [Link to external pages](#link-to-external-pages)
-- [Link to page headings](#link-to-page-headings)
 
-Although these other types of links still function, replace them with full URLs:
+You may want to link to a heading within a page.
+To do so, use one of the preceding options and then refer to [Link to page headings](#link-to-page-headings).
+
+Although these other types of links still function, replace them with one of the preceding options:
 
 - [Hugo `relref` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#relref)
+- [`docs/reference shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference)
 
 ## Link from source content that's used in multiple projects
 
-Use the `docs/reference` shortcode.
+Use `ref` URIs.
+
+{{< admonition type="note" >}}
+If you are using `doc-validator` in CI, you must upgrade to v5.0.0 to use `ref` URIs.
+{{< /admonition >}}
+
+`ref` URIs lookup destinations based upon the page's URL path and the definitions in the page's front matter.
+
+A link with a `ref` URI looks like:
+
+```markdown
+[LINK TEXT](ref:<KEY>)
+```
+
+Hugo looks up _`KEY`_ in the value for the `refs` field in the page's front matter.
+If there is no _`KEY`_ in the `refs` field, or there is no `refs` field in the front matter, Hugo logs a build error.
 
-The source is reused as described in [Reuse directories of content with Hugo mounts](https://grafana.com/docs/writers-toolkit/write/reuse-content/reuse-directories/).
-For more information and examples, refer to [`docs/reference` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsreference).
+{{< docs/shared source="writers-toolkit" lookup="refs-example.md" leveloffset="+2" >}}
```

Comment:
Given that this example text will be here, it seems like lines 44-53 would be redundant. I'll have to build to check, but just wanted to get this review in before this got merged.

Reply:
Okay I see what you're trying to do with this section. In preview, it's a hard read, so lemme think on how we might reformat it a bit and circle back.